eclass/bitcoincore.eclass

```diff
@@ -159,7 +162,7 @@ bitcoincore_common_depend_use() {
 	in_bcc_iuse "$1" || return
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $1? ( $2 )"
 }
-bitcoincore_common_depend_use upnp net-libs/miniupnpc
+bitcoincore_common_depend_use upnp '>net-libs/miniupnpc-1.9.20150915'
```

Comment:
https://github.com/gentoo/gentoo/pull/214#issuecomment-148698317 mgorny indicated that the new release does not fix the security bug